app/src/main/java/cm/aptoide/pt/install/InstallManager.java

```diff
@@ -562,4 +574,37 @@ public boolean wasAppEverInstalled(String packageName) {
         .toBlocking()
         .first();
   }
+
+  public void moveCompletedDownloadFiles(Download download) {
+    for (final FileToDownload fileToDownload : download.getFilesToDownload()) {
+      Logger.getInstance()
+          .d("AptoideDownloadManager", "trying to move file : "
+              + fileToDownload.getFileName()
+              + " "
+              + fileToDownload.getPackageName());
+      if (FileUtils.fileExists(fileToDownload.getFilePath())) {
+        String newFilePath = getFilePathFromFileType(fileToDownload);
+        fileUtils.copyFile(cachePath, newFilePath, fileToDownload.getFileName());
```

Comment:
The copy should be from the folder where the file is already (so from fileToDownload.getFilePath())
Also, to avoid moving the same file from one folder to the same, we should make a verification if it is different than the newPath